src/code/providers/readonly-provider.coffee

```diff
@@ -51,7 +57,17 @@ class ReadOnlyProvider extends ProviderInterface
       # clone the metadata items so that any changes made to the filename or content in the edit is not cached
       callback null, _.map items, (metadataItem) -> new CloudMetadata metadataItem
 
-  canOpenSaved: -> false
```

Comment:
It's not strictly necessary, because the ProviderInterface by default returns `true`, so, since we extend it, we return true as well.

I didn't edit the ProviderInterface (it's always been that way, I guess), but it's true that it's ambiguous. Especially since the default `getOpenSavedParams` in ProviderInterface is `notImplemented`.

I think I should change the default to return `false`, and add this back here.